applications/luci-app-acme/htdocs/luci-static/resources/view/acme.js

```diff
@@ -17,12 +17,15 @@ return view.extend({
 				return certs;
 			}),
 			L.resolveDefault(fs.stat('/usr/lib/acme/client/dnsapi'), null),
+			fs.lines('/proc/sys/kernel/hostname'),
```

Comment:
Shouldn't this be taken from the uci system config instead of reading it from the kernel?

Reply:
Also, how does this work with the luci acl list (regardless of whether we're reading it from the kernel or from uci)?

Reply:
Hmm, actually, even if the router does have a FQDN assigned, the system hostname is usually just the local part of it. So I'm not sure this hack is worth having at all...